Parsers/ASimAuthentication/Parsers/ASimAuthenticationAADManagedIdentity.yaml

```diff
@@ -1,19 +1,19 @@
 Parser:
-  Title: Azure active directory managed identity authentication
+  Title: Authentication ASIM parser for Microsoft AAD managed identity authentication
   Version: '0.1.0'
   LastUpdated: June 17, 2021
 Product:
-  Name: Azure active directory managed identity signin logs
+  Name: Microsoft AAD (managed identity signin logs)
 Normalization:
   Schema: Authentication
   Version: 0.1.0'
 References:
 - Title: ASIM Authentication Schema
   Link: https://aka.ms/ASimAuthenticationDoc
 - Title: ASIM
-  Link: https:/aka.ms/AboutASIM
+  Link: https://aka.ms/AboutASIM
 Description: |
-  This Query Parser maps Azure Active Directory Managed Identity sign in logs (AADManagedIdentitySignInLogs) to the Advanced Security Information Model authenticaion schema.
+  This ASIM parser supports normalizing Microsoft AAD managed identity logs, produced by the Microsoft AAD connector, to the ASIM Authentication normalized schema.
```

Comment:
"managed identity logs" -> "managed identity sign-in logs"

Reply:
ignoring. Accepting the other comment